scripts/build-ext-v4.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file Build Safari extension resources using the Vite JavaScript API
+ * @see {@link https://vitejs.dev/guide/api-javascript.html|JavaScript API}
```

Comment:
```suggestion
 * @see {@link https://vitejs.dev/guide/api-javascript.html JavaScript API}
```

Remove `|` to make links clickable in VSCode.